recipes/src/main/resources/quarkus-updates/core/3.3.yaml

```diff
@@ -0,0 +1,33 @@
+#####
+# Adjust properties in application.properties
+#####
+---
+type: specs.openrewrite.org/v1beta/recipe
+name: io.quarkus.updates.core.quarkus33.ApplicationProperties
+applicability:
+  singleSource:
+    - org.openrewrite.FindSourceFiles:
+        filePattern: "**/application*.properties"
+recipeList:
+  - org.openrewrite.properties.ChangePropertyKey:
+      oldPropertyKey: quarkus.hibernate-search-orm.automatic_indexing.synchronization.strategy
+      newPropertyKey: quarkus.hibernate-search-orm.indexing.plan.synchronization.strategy
+  - org.openrewrite.properties.DeleteProperty:
+      propertyKey: quarkus.hibernate-search-orm.automatic_indexing.enable_dirty_check
```

Comment:
That's a breaking change though... the property is still supported, it's just deprecated.
Shouldn't we rather log a warning or something?